tox.ini

```diff
@@ -170,7 +170,7 @@ deps =
     ansible
     jmespath
 commands =
-    bash {toxinidir}/.travis/runcollection.sh {toxworkdir} master 'black,flake8,yamllint,py38'
+    bash {toxinidir}/.travis/runcollection.sh {toxworkdir} main 'black,flake8,yamllint,py38'
```

Comment:
This needs to be fixed in the CI template.

Reply:
@nhosoi ^^^
* need to make the runcollection.sh script detect if the default main branch is `main` or `master` - maybe use `main` if it exists, otherwise, use `master` (if `STABLE_TAG` is unset)
* need to merge the shellcheck PR into network so that we can use the default envlist rather than having a custom envlist here to workaround the missing shellcheck PR . . .

Reply:
Correction: the branch name `master` is for downloading lsr_role2collection.py. So, there is no need to replace it with main. I'm just working on the second item from Rich.

Reply:
hmm - well if the network role is trying to remove every trace of exclusive language, then the use of `master` here might be problematic . . . but this particular problem goes away once we have tox-lsr

Reply:
Then, can we leave it for now?

Reply:
I've opened a [pr](https://github.com/linux-system-roles/network/pull/307) to sync with the template, which eliminates the master option.

Please note that the default branch name for the lsr_role2collection.py script in .travis/runcollection.sh remains. I hope it's ok.
https://github.com/linux-system-roles/network/pull/307/files#diff-91dffa6ccef3dee4f1536129fe5b9d5b9c87eb357ba4f241b6d1daa6f230a3adR16

Reply:
Would it be possible to change the default branch name for the script to main?

Reply:
Thank you! I've removed my changes. In addition it would be very nice to rename all "master" branchs to "main".